data/korath/korath missions.txt

```diff
@@ -257,3 +257,337 @@ mission "Small Scale Delivery"
+	source "Laki Nemparu"
+	destination "Seleptra Nak"
+	cargo "Efreti coffins" 13
+	landing
+
+	to offer
+		has "Efreti: Distress Call 2: done"
+
+	on offer
+		conversation
+			`You are relaxing at a spaceport restaurant now that the evacuation is complete when a delegation of Korath approach you somewhat hesitantly. They are dressed in what appears to be much finer clothing than most of the Korath you have met, certainly nothing you would wear to do work in a spaceport. As they make the traditional gesture of greeting, which you return, you are curious both as to what they want and how they will ask.`
+			`	One of them takes a small case out of a pocket and removes a black device that looks like three triangles hooked together. He hisses at it for a moment and then it rises out of his hand. The air around it begins to shimmer and soon a life size hologram of a Quarg appears.`
+			`	"Salutations, far traveler," the Quarg rumbles. "These Korath-Friends are the princes of the clan-company that you have assisted. They thank you for your aid in saving their children and grandchildren. While most grateful they are for the living, they cannot fully mourn the dead who lie on <planet>. They ask me to ask that you travel there and bring their lost ones home to <origin>."`
+			choice
+				`	"I will bring them home."`
+					goto home
+				`	"I will if they pay me."`
+					goto pay
+				`	"I won't risk my ship for the dead."`
+			`	You see the leader of the group visibly sag as the Quarg translates your words. Two younger Korath bear him up as the group turns and leaves.`
+				decline
+
+			label home
+			`	You hadn't realized how tense the group was until you saw them relax as your words reach them. They seem relieved at your answer.`
+				goto title
+
+			label pay
+			`	The group relaxes quickly as you see ripple of relief and pride pass through the group.`
+
+			label title
+			`	The leader turns and speaks to you directly. His eyes never waver from yours even as you wait for the translation. "Efreti crews have families, as children learn by working with parents. For some families, three generations were lost. We will not try to salvage the ship, for none can bear to fly it. We offer you title to salvage the <EDC-ship> and carry our grief far from here."`
+				accept
+	npc
+		system "Seketra"
+		fleet "Small Kor Mereti"
+		government "Kor Mereti"
+		to spawn
+			random < 30
+		personality
+			staying
+
+mission "Efreti: Distress Call 4"
+	landing
+	name "Recover the fallen of <EDC-ship>"
+	description "Return the bodies of the dead <EDC-ship> crew to <destination>."
+	source "Seleptra Nak"
+	destination "Karek Fornati"
+	cargo "Efreti deceased" 13
+	to offer
+		has "Efreti: Distress Call 3: done"
+	on offer
+		conversation
+			`You enter the <EDC-ship> wearing environment suits. It's been several days since the crash and Korath ships run hot. This will be an unpleasant task in the suits and perhaps impossible without them.`
```

Comment:
```suggestion
			`Locating the <EDC-ship>, you and your crew enter it wearing environment suits. It's been several days since the crash, and Korath ships run hot. This will be an unpleasant task in the suits and perhaps impossible without them.`
```

Changing excessive use of "You...", and this helps lead into the situation rather than you landing on the planet and are suddenly entering the ship, especially when first locating it was difficult. Adding the crew part is important when referring to "suits".